tools/plot.py

```diff
@@ -353,3 +351,132 @@ def plot_msd(job, start_time_fit_fs=None, ax=None):
     ax.set_title("MSD: Diffusion coefficient = {:.2e} m^2/s".format(diffusion_coefficient))
 
     return ax
+
+
+def plot_work_function(
```

Comment:
Does this need an example update to include this?

Reply:
There are two tests, but in AMS: $AMSHOME/examples/QE/WorkFunction/, $AMSHOME/examples/QE/WorkFunction/DipoleCorrection_WorkFunction